.drone.yml

```
@@ -402,6 +402,17 @@ pipeline:
       matrix:
         TEST_SUITE: cli
 
+  local-cli-acceptance-tests:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    environment:
+      - TEST_SERVER_URL=${SERVER_PROTOCOL}://server-${SERVER_PROTOCOL}
+    commands:
+      - make local-test-acceptance-cli
```

```suggestion
      - make test-acceptance-local-cli
```
All the acceptance test make targets should start with `test-acceptance` so that they are consistent and you can auto-complete nicely when typing.

IMO `local-cli` is an OK name. Others might have other ideas?

---

I'd opt for using `TESTING_EXTERNAL` env variable instead of creating targets from makefile, and set that env variable and use tag (eg: `@local-cli`) and skip-tag (`skipWhenTestingExternal` or something similar).

---

> I'd opt for using TESTING_EXTERNAL env variable instead of creating targets from makefile

For that we have to remove `--remote` flag from test start script in our `Makefile`.

And I think think that is a good way to do this.
What do you think @phil-davis?